src/adapter/src/coord/sequencer/inner/cluster.rs

```diff
@@ -70,23 +68,17 @@ impl Coordinator {
         ctx: ExecuteContext,
         plan: plan::AlterClusterPlan,
     ) {
-        let stage = return_if_err!(self.alter_cluster_validate(ctx.session(), plan).await, ctx);
+        let stage = return_if_err!(self.alter_cluster_validate(plan).await, ctx);
         self.sequence_staged(ctx, Span::current(), stage).await;
     }
 
     #[instrument]
     async fn alter_cluster_validate(
         &mut self,
-        session: &Session,
         plan: plan::AlterClusterPlan,
     ) -> Result<ClusterStage, AdapterError> {
-        let validity = PlanValidity::new(
-            self.catalog().transient_revision(),
-            BTreeSet::new(),
-            Some(plan.id.clone()),
-            None,
-            session.role_metadata().clone(),
-        );
+        let validity =
+            PlanValidity::require_transient_revision(self.catalog().transient_revision());
```

Comment:
I thought that we weren't serializing `ALTER CLUSTER`?

Reply:
Good catch. Reverted.